pkg/api/mlflow/service/artifact/storage/gs.go

```diff
@@ -25,7 +27,19 @@ type GS struct {
 }
 
 // NewGS creates new GC instance.
-func NewGS(ctx context.Context, _ *config.ServiceConfig) (*GS, error) {
+func NewGS(ctx context.Context, config *config.ServiceConfig) (*GS, error) {
+	if config.GSEndpointURI != "" {
+		client, err := storage.NewClient(
+			ctx, option.WithEndpoint(config.GSEndpointURI), option.WithoutAuthentication(),
```

Comment:
@jgiannuzzi the only one concern I can see here -> when we use custom endpoint we need an option to disable `authentication`. So we have 2 ways:
- just hardcode it as is - because only the case when we need to use `custom` endpoint is `tests`. nobody else will do that in any other case.
- we can of course make it configurable via additional flag. so for example -> `FML_GS_DISABLE_AUTH` so in that case everything  will be super configurable.
what do you think? My choose is 1 =) until we really need it.

Reply:
I agree with you on 1 (+ some comments explaining why it's hardcoded)
But here is another idea: let's reuse our `config.devMode` setting for enabling `option.WithoutAuthentication()`!

Reply:
hm, ok!. let me quickly fix it!

Reply:
oook. have to revert =)

Reply:
done.